tests/unit/test_pep621.py

```diff
@@ -0,0 +1,48 @@
+# This file is part of CycloneDX Python
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OWASP Foundation. All Rights Reserved.
+
+import os
+import tempfile
+import unittest
+
+from cyclonedx.factory.license import LicenseFactory
+from cyclonedx.model.license import DisjunctiveLicense, LicenseAcknowledgement
+
+from cyclonedx_py._internal.utils.pep621 import project2licenses
+
+
+class TestPEP621(unittest.TestCase):
+    def setUp(self):
+        self.lfac = LicenseFactory()
```

Comment:
isee foreign factories being not mocked, but used. this makes the tests not unittests but integrationtests.
please move the file in the `unit` folder into the `integration` folder.